cli_tools/gce_vm_image_import/importer/disk_inspection_processor.go

```diff
@@ -41,8 +41,13 @@ func (p *diskInspectionProcessor) process(pd persistentDisk,
 		return pd, err
 	}
 
-	pd.isUEFICompatible = p.args.UefiCompatible || ir.HasEFIPartition
-	loggableBuilder.SetUEFIMetrics(pd.isUEFICompatible, ir.HasEFIPartition)
+	isHybrid := ir.UEFIBootable && ir.BIOSBootable
+	if !p.args.UefiCompatible && isHybrid {
+		log.Printf("This disk can boot with either BIOS or a UEFI bootloader. The default setting for booting is BIOS. " +
+			"If you want to boot using UEFI, please reference to https://cloud.google.com/compute/docs/import/importing-virtual-disks#importing_a_virtual_disk_with_uefi_bootloader'.")
```

Comment:
```suggestion
		log.Printf("This disk can boot with either BIOS or a UEFI bootloader. The default setting for booting is BIOS. " +
			"If you want to boot using UEFI, please see https://cloud.google.com/compute/docs/import/importing-virtual-disks#importing_a_virtual_disk_with_uefi_bootloader'.")
```

Reply:
"reference to" is not from Shawnique, let me change to "see"